YesterdayApi/Utilities/Exceptions/ObjectNotFoundException.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Net;
+using System.Runtime.Serialization;
+
+namespace YesterdayApi.Utilities.Exceptions
+{
+    [Serializable]
```

Comment:
Why do we Serializable?

Reply:
Microsoft uses [Serializable] on every custom exception and also we need it to be able to serialize it into Json to send it over the web request.( here's a link to a helpful SO answer https://stackoverflow.com/questions/1066701/why-should-i-always-make-my-exceptions-serializable-net )